website/docs/r/search_index.html.markdown

```diff
@@ -0,0 +1,340 @@
+            * `first`
+
+      If `matches` is set to `all, replace all matching patterns. Otherwise, replace only the first matching pattern.
+    * [snowballStemming](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-snowballstemming-tf-ref) - Stems tokens using a [Snowball-generated stemmer](https://snowballstem.org/).
+        * `type` - Must be `snowballstemming`
+        * `stemmer_name` - (Required) The following values are valid:
+            * `arabic`
+            * `armenian`
+            * `basque`
+            * `catalan`
+            * `danish`
+            * `dutch`
+            * `english`
+            * `finnish`
+            * `french`
+            * `german`
+            * `german2` (Alternative German language stemmer. Handles the umlaut by expanding ü to ue in most contexts.)
+            * `hungarian`
+            * `irish`
+            * `italian`
+            * `kp` (Kraaij-Pohlmann stemmer, an alternative stemmer for Dutch.)
+            * `lithuanian`
+            * `lovins` (The first-ever published "Lovins JB" stemming algorithm.)
+            * `norwegian`
+            * `porter` (The original Porter English stemming algorithm.)
+            * `portuguese`
+            * `romanian`
+            * `russian`
+            * `spanish`
+            * `swedish`
+            * `turkish`
+    * [stopword](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-stopword-tf-ref) - Removes tokens that correspond to the specified stop words. This token filter doesn't analyze the specified stop word:
+        * `type` - Must be `stopword`
+        * `token` - (Required) The list of stop words that correspond to the tokens to remove. Value must be one or more stop words.
+        * `ignore_case` - The flag that indicates whether or not to ignore case of stop words when filtering the tokens to remove. The value can be one of the following:
+            * `true` - to ignore case and remove all tokens that match the specified stop words
+            * `false` - to be case-sensitive and remove only tokens that exactly match the specified case
+
+      If omitted, defaults to `true`.
+    * [trim](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-trim-tf-ref) - Trims leading and trailing whitespace from tokens.
+
+
+
+> **NOTE:** Do not use [daitchMokotoffSoundex](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-daitchmokotoffsoundex-tf-ref) token filter in operators where `fuzzy` is enabled. Atlas Search supports the `fuzzy` option for the following operators:
+>* [autocomplete](https://docs.atlas.mongodb.com/reference/atlas-search/autocomplete/#std-label-autocomplete-ref)
+>* [term (Deprecated)](https://docs.atlas.mongodb.com/reference/atlas-search/term/#std-label-term-ref)
+>* [text](https://docs.atlas.mongodb.com/reference/atlas-search/text/#std-label-text-ref)
+
+
+
+## Attributes Reference
```

Comment:
I think we are missing these?

Reply:
not really, I made a lot of mistakes when copied document from another file. The attributes reference are the same  as above